scm/etc/scripts/MOSAiC_AMPS_forcing_file_generator.py

```diff
@@ -0,0 +1,590 @@
+h_advec_thil = nc_fid.variables['h_advec_thetail'][:] #K/s
+h_advec_thil = np.swapaxes(h_advec_thil, 0, 1) #swap the time and levels axis
+#for t in range(time.size):
+#    h_advec_thil[:,t] = (ffc.p0/levels)**(ffc.R_dry/ffc.c_p)*h_advec_T[:,t] #convert to potential temperature
+
+#v_advec_thil = np.zeros((levels.size,time.size),dtype=float)
+v_advec_thil = nc_fid.variables['v_advec_thetail'][:] #K/s
+v_advec_thil = np.swapaxes(v_advec_thil, 0, 1) #swap the time and levels axis
+#for t in range(time.size):
+#    v_advec_thil[:,t] = (ffc.p0/levels)**(ffc.R_dry/ffc.c_p)*v_advec_T[:,t] #convert to potential temperature
+
+dT_dt = np.zeros((levels.size,time.size),dtype=float)
+
+# h_advec_T = h_advec_T*86400.0
+# v_advec_T = v_advec_T*86400.0
+# dT_dt = dT_dt*86400.0
+# spr.contour_plot_firl(time, levels, h_advec_T, np.min(h_advec_T), np.max(h_advec_T), 'h advec T', 'time', 'pressure', 'h_advec_T.eps', y_inverted = True)
+# spr.contour_plot_firl(time, levels, v_advec_T, np.min(v_advec_T), np.max(v_advec_T), 'v advec T', 'time', 'pressure', 'v_advec_T.eps', y_inverted = True)
+# spr.contour_plot_firl(time, levels, dT_dt, np.min(dT_dt), np.max(dT_dt), 'total T tend', 'time', 'pressure', 'dT_dt.eps', y_inverted = True)
+
+#h_advec_qt = np.zeros((levels.size,time.size),dtype=float)
+h_advec_qt = nc_fid.variables['h_advec_qt'][:] #kg/kg/s
+h_advec_qt = np.swapaxes(h_advec_qt, 0, 1) #swap the time and levels axis
+h_advec_qt = h_advec_qt/(1.0 + qt_mr)**2 #convert to specific humidity
+
+h_advec_qi = np.zeros((levels.size,time.size),dtype=float)
+h_advec_ql = np.zeros((levels.size,time.size),dtype=float)
+
+#v_advec_qt = np.zeros((levels.size,time.size),dtype=float)
+v_advec_qt = nc_fid.variables['v_advec_qt'][:] #kg/kg/s
+v_advec_qt = np.swapaxes(v_advec_qt, 0, 1) #swap the time and levels axis
+v_advec_qt = v_advec_qt/(1.0 + qt_mr)**2 #convert to specific humidity
+
+dq_dt = np.zeros((levels.size,time.size),dtype=float)
+#dq_dt = nc_fid.variables['dqdt'][:]
+#dq_dt = np.swapaxes(dq_dt, 0, 1)*1.0E-3/3600.0 #swap the time and levels axis, convert to K/s
+
+#h_advec_qt = h_advec_qt*86400.0
+#v_advec_qt = v_advec_qt*86400.0
+#dq_dt = dq_dt*86400.0
+#spr.contour_plot_firl(time, levels, h_advec_qt, np.min(h_advec_qt), np.max(h_advec_qt), 'h advec q', 'time', 'pressure', 'h_advec_q.eps', y_inverted = True)
+#spr.contour_plot_firl(time, levels, v_advec_qt, np.min(v_advec_qt), np.max(v_advec_qt), 'v advec q', 'time', 'pressure', 'v_advec_q.eps', y_inverted = True)
+#spr.contour_plot_firl(time, levels, dq_dt, np.min(dq_dt), np.max(dq_dt), 'total q tend', 'time', 'pressure', 'dq_dt.eps', y_inverted = True)
+
+#phi_sfc = nc_fid.variables['phis'][:]
+#z_sfc = nc_fid.variables['alt'][:]
+height = nc_fid.variables['height'][:]
+#height = ffc.get_height_from_pres(T_abs[:,0],levels,z_sfc)
+
+#the following variables are not in this forcing file, but are included in other cases
+soil_depth = np.zeros(4,dtype=float)
```

Comment:
@hertneky It's not showing as changed because this file is new, but I added this (needed by the dephy_converter.py script when lsm_ics = T)